oxd-server/src/main/java/org/gluu/oxd/server/mapper/RegisterRequestMapper.java

```diff
@@ -0,0 +1,410 @@
+package org.gluu.oxd.server.mapper;
+
+import com.google.common.base.Strings;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.math.NumberUtils;
+import org.gluu.oxauth.client.RegisterRequest;
+import org.gluu.oxauth.model.common.AuthenticationMethod;
+import org.gluu.oxauth.model.common.GrantType;
+import org.gluu.oxauth.model.common.ResponseType;
+import org.gluu.oxauth.model.common.SubjectType;
+import org.gluu.oxauth.model.crypto.encryption.BlockEncryptionAlgorithm;
+import org.gluu.oxauth.model.crypto.encryption.KeyEncryptionAlgorithm;
+import org.gluu.oxauth.model.crypto.signature.SignatureAlgorithm;
+import org.gluu.oxauth.model.register.ApplicationType;
+import org.gluu.oxd.server.service.Rp;
+
+import java.util.stream.Collectors;
+
+public class RegisterRequestMapper {
```

Comment:
create private constructor to forbid instance creation. We do not keep internal state in this class.

Reply:
Changes done